main/core.js

```diff
@@ -189,14 +192,14 @@ async function maybeMigrateFiles () {
   } catch {
     return
   }
-  console.log(
+  log.info(format(
     'Migrating files from %s to %s',
     oldSaturnDir,
     newSaturnDir
-  )
+  ))
```

Comment:
I think we can remove `maybeMigrateFiles` now that Station no longer ships Saturn L2 Node. WDYT, @juliangruber? I can open a follow-up pull request.

Reply:
yes please! good spot

Reply:
https://github.com/filecoin-station/desktop/pull/1668